api/v1beta1/lmevaljob_types.go

```diff
@@ -59,6 +60,17 @@ type Arg struct {
 	Value string `json:"value,omitempty"`
 }
 
+type EnvSecret struct {
```

Comment:
One thing that some users may want is the ability to read secrets from files instead of the env. Would it be possible to add this as an option?

Reply:
I created another data struct to store the file-type secret. And I think it's different from env secret. So I didn't create env vars for those file-type secrets. How to use env or file secrets depends on the evaluation job. We can support two of them separately for now.